common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -501,3 +515,754 @@ def test_library_export_side_bar_help(self):
             href=href,
             help_text='Learn more about exporting a library'
         )
+
+
+class NewProgramHelpTest(ProgramsConfigMixin, WebAppTest):
+    """
+    Test help links on a 'New Program' page
+    """
+    def setUp(self):
+        super(NewProgramHelpTest, self).setUp()
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.program_page = DashboardPageWithPrograms(self.browser)
+        self.auth_page.visit()
+        self.set_programs_api_configuration(True)
+        self.program_page.visit()
+
+    def test_program_create_nav_help(self):
+        """
+        Scenario: Help link in navigation bar is working on 'New Program' page
+        Given that I am on the 'New Program' page
+        And I want help about the process
+        And I click the 'Help' in the navigation bar
+        Then Help link should open.
+        And help url should end with 'index.html'
+        """
+        self.program_page.click_new_program_button()
+        href = 'http://edx.readthedocs.org/projects/edx-partner-course-staff' \
+               '/en/latest/index.html'
+
+        # Assert that help link is correct.
+        assert_nav_help_link(
+            test=self,
+            page=self.program_page,
+            href=href,
+        )
+
+
+class CourseOutlineHelpTest(StudioCourseTest):
+    """
+    Tests help links on course outline page.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(CourseOutlineHelpTest, self).setUp()
+        self.course_outline_page = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.course_outline_page.visit()
+
+    # This scenario depends upon TNL-5460
+    # def test_course_outline_nav_help(self):
+    #     """
+    #     Scenario: Help link in navigation bar is working on Course Outline page
+    #     Given that I am on the Course Outline page
+    #     And I want help about the process
+    #     And I click the 'Help' in the navigation bar
+    #     Then Help link should open.
+    #     And help url should end with 'developing_course/course_outline.html'
+    #     """
+    #     href = 'http://edx.readthedocs.org/projects/edx-partner-course-staff' \
+    #            '/en/latest/developing_course/course_outline.html'
+    #
+    #     # Assert that help link is correct.
+    #     assert_nav_help_link(
+    #         test=self,
+    #         page=self.course_outline_page,
+    #         href=href
+    #     )
```

Comment:
@Muddasser Instead of commenting the test skip it as described in https://docs.python.org/2/library/unittest.html#unittest.skip